CMakeLists.txt

```diff
@@ -111,8 +111,8 @@ if(NOT DEFINED PLATFORM_SYSTEM_LIBRARIES)
 endif()
 
 if(WIN32)
-    # We must explicitly link to the core win32 libraries
-    list(APPEND PLATFORM_SYSTEM_LIBRARIES kernel32 user32 gdi32 winspool comdlg32 advapi32 shell32 ole32 oleaut32 uuid ws2_32 Crypt32)
+    # Core win32 libraries come with CMAKE_C(XX)_IMPLICIT_LINK_LIBRARIES
+    list(APPEND PLATFORM_SYSTEM_LIBRARIES ws2_32 crypt32)
```

Comment:
Where are these link libs used?
I ran another build in vcpkg CI with ws2_32 crypt32 removed, and it suceeded in all CRT and library linkage configurations configurations.